engine/engine/src/engine.cpp

```diff
@@ -1531,6 +1537,12 @@ namespace dmEngine
                         return;
                     }
                 }
+
+                {
+                    DM_PROFILE("Job Thread");
```

Comment:
I'd say you can put it in the system itself? (profile.h is in dlib)
Also for each worker threads.